### ISSUE TITLE
SPC w D binding does not appear in the which-key buffer

### ISSUE THREAD

**Issue by dubnde** [syl20bnr/spacemacs#5735]
#### Description
`SPC w D` does not appear in the `which-key` buffer. It actually works but does appear in the popup

#### Reproduction guide
- Start Emacs
- `SPC w` / to split window
- `SPD w` to show the popup buffer

*Observed behaviour:*
Expected to see `SPD w D` binding to appear in the `which-key` buffer

*Expected behaviour:*
`SPD w D` appears in the `which-key` buffer

#### System Info
- OS: darwin
- Emacs: 24.5.1
- Spacemacs: 0.105.16
- Spacemacs branch: develop (rev. e694853)
- Graphic display: t
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
((auto-completion :variables auto-completion-tab-key-behavior 'cycle :disabled-for shell)
 better-defaults
 (c-c   :variables c-c  -enable-clang-support t c-c  -default-mode-for-headers 'c  -mode)
 emacs-lisp evil-cleverparens evil-commentary
 (evil-snipe :variables evil-snipe-enable-alternate-f-and-t-behaviors t)
 git gtags markdown org osx semantic
 (shell :variables shell-enable-smart-eshell t)
 shell-scripts spacemacs-helm spell-checking syntax-checking version-control vim-powerline windows-scripts yaml dc-org)
```

#### Backtrace
```
BACKTRACE IF RELEVANT
```

**Comment by NJBS** [comment]
A few notes:

`spacemacs-ui` [additionally binds](https://github.com/syl20bnr/spacemacs/blob/develop/layers/%2Bspacemacs/spacemacs-ui/packages.el#L74) `ace-delete-window` to `SPC w C` yet neither `SPC w C` nor `SPC w D` show up in which-key.

Removing the `SPC w C` binding from `spacemacs-ui` fixes the issue. However, which-key should work with functions assigned to multiple bindings.

Perhaps @justbur can shed some light?

**Comment by justbur** [comment]
Does it show up in `SPC ?`?

**Comment by NJBS** [comment]
Yes, but the formatting looks a bit funky:

![example](http://i.imgur.com/CbS056U.png)

**Comment by justbur** [comment]
Ah it's a character range because C and D are next to each other. Is it
really necessary to bind to both?

**Comment by TheBB** [comment]
Can confirm it shows up if I unbind `SPC wC`. :joy:

**Comment by TheBB** [comment]
Fixed by #5739, although I assume this is also a bug in which-key.

**Comment by d12frosted** [comment]
Fixed with released of Spacemacs v0.200. Let us know if you still have any problems with this issue.
